lib/smart_answer_flows/student-finance-calculator/student_finance_calculator.govspeak.erb

```diff
@@ -21,9 +21,5 @@
 <% end %>
 
 <% content_for :post_body do %>
-  If you're a part-time student applying for the 2018 to 2019 academic year, you'll need to know how many credits you'll study and how many credits a full-time student would study.
-
-  Ask your university or college if you're not sure.
```

Comment:
This and the sentence above weren't detailed in the spreadsheet, but didn't make sense now we've removed the 2018-19 dates. Should they instead be updated to reference more recent years, or to remove the "applying for ... academic year" portion?

Reply:
Hey Issy, you're right. Good spot. Looks to me like this should have been removed last year as that's the case whatever year you're applying. Please update so sentence reads:

"If you’re a part-time student you’ll need to know how many credits you’ll study and how many credits a full-time student would study."

Reply:
Thanks. Done!